### PR TITLE
Tickets/dm 34754

### PR DESCRIPTION
**Pull request by leannep**

### CODE REVIEW

File: body.tex [hunk 1]
```diff
@@ -1,21 +1,341 @@
...
+
+During LSST observing, early analysis of the images coming off the camera will be carried out on the mountain on the timescales of a few minutes. 
+The following are examples of metrics that could be implemented in \faro to support this rapid first-look analysis:
+\begin{itemize}
+\item Number of detected Objects,
+\item Sky brightness, counts and variance 
+\item Image quality, specifically FWHM, PSF ellipticity
+\item Comparison to reference catalogs, e.g Gaia to look for an excess/deficit of detections, astrometric offsets, photometric residuals or problems with PSF modelling,
+\item Effective survey depth
+\item Anomoly detection
```
Comment (ctslater) [inline body.tex]: Anomaly

File: body.tex [hunk 2]
```diff
@@ -1,21 +1,341 @@
...
+\caption[short]{The distributions of RMS values of the photometric repeatability of sources detected as part of the initial AuxTel imaging campaign. Vertical dashed lines show the calculated PA1 metric for each bandpass. The left panel shows the value of PA1 calculated for all sources detected and the right panel includes only the sources in the central region of the detector (1000 < X,Y < 3000 pixels). The central region of the detector is not affected by the vignetting due the epoxy creep on the filters leading to much tighter distributions and lower PA1 values.}
+\label{fig:faro_auxtel_metrics}
+\end{figure}
+
+\subsection{Rapid and first-look analysis} \label{ssec:rapid}
+
+During LSST observing, early analysis of the images coming off the camera will be carried out on the mountain on the timescales of a few minutes. 
+The following are examples of metrics that could be implemented in \faro to support this rapid first-look analysis:
+\begin{itemize}
+\item Number of detected Objects,
```
Comment (ctslater) [inline body.tex]: Sounds a little trivial

File: body.tex [hunk 3]
```diff
@@ -1,21 +1,341 @@
...
+      python: |
+        from lsst.faro.measurement import PF1Task
+        config.measure.retarget(PF1Task)
+        config.measure.threshPA2 = 15.0
+    \end{lstlisting}
+  \caption{A \faro pipeline describing the measurement stage of the computation of three performance metrics per-tract from matched catalogs.}
+  \label{fig:faro_pipeline}
+\end{figure}
+
+\begin{figure}[!ht]
```
Comment (ctslater) [inline body.tex]: I think the description in the text is sufficient for describing the organization, and this snippet of yaml doesn't really add much.

File: abstract.tex [hunk 4]
```diff
@@ -1,18 +1,12 @@
 \begin{abstract}
 
-The Vera C. Rubin Observatory will advance nearly every area of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
-The LSST will produce approximately 20TB of raw data per night, which will be automatically processed by the LSST Science Pipelines 
-to generate science-ready data products -- processed images, catalogs and alerts. 
+The Vera C.\ Rubin Observatory will advance nearly every area of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST) \cite{2019ApJ...873..111I}.
```
Comment (ctslater) [inline abstract.tex]: "nearly every area of astronomy" seems rather strong.

**Review by ctslater — APPROVED** [review]

File: body.tex (hunk 2 above)
Comment (leannep) [reply body.tex]: removed

File: body.tex (hunk 3 above)
Comment (leannep) [reply body.tex]: removed

File: abstract.tex (hunk 4 above)
Comment (leannep) [reply abstract.tex]: nearly every area-> many areas

**Pull request merged**